### DISCUSSION TITLE
Creating a new repository to host pre-processing functions

### DISCUSSION THREAD

**Discussion by exook** [baler-collaboration/baler#113] [Ideas]
Pre-processing of HEP data does not really belong in baler. Implementing pre-processing in the configuration file does not prove to be a more elegant solution (see figure). Baler should require that the user provides the data clean as a pickled pandas dataframe, how to get there is up to the user.

The problem with this for our use case is that each HEP file we use will require slightly different pre-processing, and we don't want these scripts to exist only locally. 

TL;DR: Pre-processing functions should not come with Baler, but we want to store and share our functions somewhere safe (on git?)

![image](https://user-images.githubusercontent.com/23464789/220140262-c00c13c1-fb2f-4a15-9243-0c0c07acfbb3.png)

**Comment by exook** [comment]
I think we could create a second repository under the "Baler" organization for pre-processing of data before running it through Baler. Thoughts?

@fritjof-b @OliverWoolland @PRAkTIKal24 @gallenaxel
